Mathlib/Algebra/Group/Pi/Lemmas.lean

```diff
@@ -305,17 +265,6 @@ theorem MonoidHom.single_apply [∀ i, MulOneClass <| f i] (i : I) (x : f i) :
 #align monoid_hom.single_apply MonoidHom.single_apply
 #align add_monoid_hom.single_apply AddMonoidHom.single_apply
 
-/-- The multiplicative homomorphism including a single `MulZeroClass`
-into a dependent family of `MulZeroClass`es, as functions supported at a point.
-
-This is the `MulHom` version of `Pi.single`. -/
-@[simps]
-def MulHom.single [∀ i, MulZeroClass <| f i] (i : I) : f i →ₙ* ∀ i, f i where
```

Comment:
Why did you rename this? Doesn't this name match `AddMonoidHom.single`?

Reply:
My reasoning was that it's better to have `Pi.single` somewhere in the name, and also it fits with the other declarations in the `Pi` namespace. But yes I see that it has many friends which I haven't renamed.

Thoughts?

Reply:
Since there's a defensible argument to not do this change, let's postpone it to a separate PR that renames all the `single`s at once. I much prefer PRs that try to do as little as possible.